Samples/SwiftKitSampleApp/Sources/MySwiftLibrary/MySwiftLibrary.swift

```diff
@@ -77,6 +91,15 @@ public class MySwiftClass {
     p("i:\(i)")
   }
 
+  // TODO: workaround until we expose properties again
```

Comment:
TODO: Seems we didn't yet finish moving to the "thunk way" the properties, gotta do that.